sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -3368,18 +3368,18 @@ def hpat_pandas_series_min(self, axis=None, skipna=None, level=None, numeric_onl
     ty_checker = TypeChecker(_func_name)
     ty_checker.check(self, SeriesType)
 
-    if not isinstance(self.data.dtype, types.Number):
+    if not isinstance(self.data.dtype, (types.Integer, types.Float)):
         raise TypingError(
             '{} Currently function supports only numeric values. Given data type: {}'.format(
                 _func_name, self.data.dtype))
 
-    if not isinstance(skipna, (types.Omitted, types.Boolean)) and skipna is not True:
+    if not (isinstance(skipna, (types.Omitted, types.Boolean, types.NoneType)) or skipna is True or skipna is None):
```

Comment:
I think, it's better to separate `skipna is True` ... from common `not`, because it's not clear here